deploy/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+metadata:
+  name: piraeus-operator
+resources:
+  - piraeus/templates/operator-controller-passphrase.yaml
+  - piraeus/templates/operator-satelliteset.yaml
+  - piraeus/templates/csi-controller-rbac.yml
+  - piraeus/templates/tests/test-CS-svc.yaml
+  - piraeus/templates/tests/test-NS-svc.yaml
+  - piraeus/templates/ha-controller-deployment.yaml
+  - piraeus/templates/operator-serviceaccount.yaml
+  - piraeus/templates/operator-csi-driver.yaml
+  - piraeus/templates/operator-controller.yaml
+  - piraeus/templates/satellite-rbac.yml
+  - piraeus/templates/operator-deployment.yaml
+  - piraeus/templates/controller-rbac.yml
+  - piraeus/templates/ha-controller-rbac.yaml
+  - piraeus/templates/csi-node-rbac.yml
+  - piraeus/charts/etcd/templates/rbac.yaml
+  - piraeus/charts/etcd/templates/service.yaml
+  - piraeus/charts/etcd/templates/statefulset.yaml
```

Comment:
When I run `make deploy/piraeus`, I get changes to this file. I think the script needs to `find ... | sort`, and then regenerate this file, so we don't get random changes to this file